app/controllers/items/search_location.js

```diff
@@ -24,6 +33,21 @@ export default searchModule.extend({
   selectedLocation: null,
   hideDetailsLink: true,
 
+  sameSingleLocation: Ember.computed("selectedLocation", function() {
+    if (this.get('item.packagesLocations').get('length') === 1){
+     return this.get('item.packagesLocations').get('firstObject').get("location_id") === parseInt(this.get('selectedLocation.id'));
```

Comment:
No need to call get twice. Instead you can do this
**this.get('item.packagesLocations').get("firstObject.location_id")** or **this.get('item.packagesLocations.firstObject.location_id')**